bin/mozilla/rp.pl

```diff
@@ -1659,7 +1658,8 @@ sub generate_ap_aging {
 		$locale->text('Invalid to date ...') );
 
 	# split vendor
-	( $form->{vendor} ) = split( /--/, $form->{vendor} );
+	( $form->{vendor}, $form->{vendor_id} ) = split( /--/, $form->{vendor} );
+    $form->{vendor_id} *= 1;
```

Comment:
why vendor_id *= 1?

Reply:
As mentioned elsewhere, makes sure it is 0 if it is non-numeric or null.